packages/driver/src/cypress/events.ts

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash'
+import { EventEmitter2 } from 'eventemitter2'
+import { debug as Debug } from 'debug'
+import * as Bluebird from 'bluebird'
```

Comment:
```suggestion
import Bluebird from 'bluebird'
```

otherwise it breaks types